design/sketch.webidl

```diff
@@ -401,13 +401,20 @@ dictionary WebGPUShaderModuleDescriptor {
 interface WebGPUShaderModule {
 };
 
-// Description of the framebuffer attachments
-dictionary WebGPUAttachmentsStateDescriptor {
-    sequence<WebGPUTextureFormatEnum> formats;
-    // TODO other stuff like sample count etc.
+// Description of a single attachment
+dictionary WebGPUAttachment {
+    // Attachment data format
+    WebGPUTextureFormatEnum format;
+    // Number of MSAA samples
+    u32 samples;
```

Comment:
This is pending discussion of multisampling, but I think we also need to add "boolean isResolvedAtEndOfRenderPass" because VkRenderPass compatibility includes "VkSubpassDescription:: pResolveAttachments". I'll open an issue on Vulkan-Docs to ask if this can be relaxed for single-subpass render passes.

Reply:
the point of this PR is that we can derive the need of those being resolved simply by looking at the sample counts